tensorflow/core/kernels/reduction_ops.h

```diff
@@ -117,6 +117,10 @@ struct Identity {
 FIX_MEAN_IDENTITY(Eigen::half)
 FIX_MEAN_IDENTITY(float)
 FIX_MEAN_IDENTITY(double)
+#if GOOGLE_CUDA
+FIX_MEAN_IDENTITY(complex64)
```

Comment:
Please revert this change.

Reply:
Reverted in https://github.com/tensorflow/tensorflow/pull/32474/commits/2118059a0f79fc14e14631663bf3fc2ecbe2bad9